src/plugins/intel_gpu/include/intel_gpu/primitives/paged_attention.hpp

```diff
@@ -13,14 +13,34 @@ namespace cldnn {
 struct paged_attention : public primitive_base<paged_attention> {
     CLDNN_DECLARE_PRIMITIVE(paged_attention)
 
+    enum PagedAttentionInputIdx {
+        QUERY = 0,
+        KEY = 1,
+        VALUE = 2,
+        KEY_CACHE = 3,
+        VALUE_CACHE = 4,
+        PAST_LENS = 5,
+        SUBSEQUENCE_BEGINS = 6,
+        BLOCK_INDICES = 7,
+        BLOCK_INDICES_BEGINS = 8,
+        SCALE = 9,
+        SLIDING_WINDOW = 10,
+        ALIBI = 11,
+        MAX_CONTEXT_LEN = 12,
+        SCORE_AGGREGATION = 13,
+        ROTATED_BLOCK_INDICES = 14,
+        ROTATION_DELTAS = 15,
+        ROTATION_TRIG_LUT = 16,
+    };
+
     static constexpr size_t block_size = 16;
 
     paged_attention() : primitive_base("", {}) {}
 
     paged_attention(const primitive_id& id,
                     const std::vector<input_info>& inputs)
         : primitive_base(id, inputs) {
-        OPENVINO_ASSERT((inputs.size() == 13) || (inputs.size() == 16),
+        OPENVINO_ASSERT((inputs.size() == 14) || (inputs.size() == 17),
```

Comment:
score_aggregation is optional but do we need 14 at least?

Reply:
Score_aggregation has become required input, but at ov::Model level it can either be Constant (scores disabled) or Parameter (containing window sizes)